templates/start-slave.sh.erb

```diff
@@ -1,4 +1,68 @@
 #!/usr/bin/env bash
 
-source <%= @defaults_location %>/jenkins-slave 
-java -jar <%= @slave_home %>/<%= @client_jar %> $JENKINS_SLAVE_ARGS
+source <%= @defaults_location %>/jenkins-slave
+
+# mandatory input vars
```

Comment:
Instead of cut'n'pasting the the entire `jenkins-slave-run` script, how about installing it and call it from this script?